extension/pgtelemetry--1.2--1.3.sql

```diff
@@ -0,0 +1,40 @@
+create or replace function vacuum_progress()
+  returns table
+  (
+    datname               name,
+    schemaname            name,
+    relname               name,
+    phase                 text,
+    heap_blks_total       int8,
+    heap_blks_scanned     int8,
+    heap_blks_vacuumed    int8,
+    index_vacuum_count    int8,
+    max_dead_tuples       int8,
+    num_dead_tuples       int8
+  )
+  language plpgsql stable as
+$fnc$
+begin
+  /* too lazy to check which version of pg the view was added in, so
```

Comment:
It's 9.6 and everything below 10 is unsupported; the check can be removed and the function can be converted to a simple view `table pg_stat_progress_vacuum`; not sure if we even need it here, perhaps pg-management can be fixed to query pg_stat_progress_vacuum directly.

Reply:
It is called as a function, so if we go that way we'd need to also change the Prometheus query. We can though, you're right.